client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -293,31 +300,53 @@ const JetpackComMasterbar: React.FC< Props > = ( { pathname } ) => {
 																	) ) }
 																</ul>
 
-																<div className="header__submenu-bundles">
-																	<hr />
+																<hr className="header__submenu-section-separator" />
+
+																<div className="header__submenu-bottom-section">
+																	<div className="header__submenu-bundles">
+																		<p className="header__submenu-category-heading">
+																			{ bundles.label }
+																		</p>
 
-																	<p className="header__submenu-bundles-heading">
-																		{ bundles.label }
-																	</p>
+																		<ul className="header__submenu-links-list">
+																			{ bundles.items.map( ( { label, tagline, href } ) => {
+																				return (
+																					<li key={ `bundles-${ href }` }>
+																						<ExternalLink
+																							className="header__submenu-link"
+																							href={ localizeUrl( href, locale ) }
+																							onClick={ onLinkClick }
+																						>
+																							<span className="header__submenu-label">
+																								{ label }
+																							</span>
+																							<span className="header__submenu-tagline">
+																								{ tagline }
+																							</span>
+																						</ExternalLink>
+																					</li>
+																				);
+																			} ) }
+																		</ul>
+																	</div>
+																	<div className="header__submenu-apps-wrapper">
+																		<p className="header__submenu-category-heading">
+																			{ appsLink.categoryLabel }
+																		</p>
 
-																	<ul className="header__submenu-links-list">
-																		{ bundles.items.map( ( { label, tagline, href } ) => {
-																			return (
-																				<li key={ `bundles-${ href }` }>
-																					<ExternalLink
-																						className="header__submenu-link"
-																						href={ localizeUrl( href, locale ) }
-																						onClick={ onLinkClick }
-																					>
-																						<span className="header__submenu-label">{ label }</span>
-																						<span className="header__submenu-tagline">
-																							{ tagline }
-																						</span>
-																					</ExternalLink>
-																				</li>
-																			);
-																		} ) }
-																	</ul>
+																		<ExternalLink
```

Comment:
Is there a reason not to use the `<ul className="header__submenu-links-list">` here as well? 
If the list structure can be re-used, would that reduce the number of new styles that we need to add as well?

Reply:
Semantically, that isn't a list ~yet~, and there's no style I could reuse from the list other than the `margin-top` applied to the `li`.

Reply:
> Semantically, that isn't a list yet

🤔 Would this be due to the number of items?
The W3C[ recommendations document for lists](https://www.w3.org/TR/html401/struct/lists.html) indicates lists must contain one or more `<li>` elements - is there a reason not to have a list with one item here?

Reply:
>🤔 Would this be due to the number of items?

Mostly because of its context. That is not a group of list items that happens to have only a single item at the moment - which would be a list by definition. 

I can definitely change it in this PR if that's what's holding the approval, tho.

Reply:
> Mostly because of its context. That is not a group of list items that happens to have only a single item at the moment - which would be a list by definition.

Ahhh I see the argument there, thanks for the additional clarity!
I see you already updated it, I think it works either way 👍